info.yml

```diff
@@ -73,6 +73,12 @@ configuration:
         default_value_tk-mari:
             Alembic Cache: [geometry_import]
 
+    entity_mappings:
+        type: dict
+        description: Associates entity types with actions. The actions are all defined
+                     inside the actions hook, so a number of actions are available by default
```

Comment:
Could you remove `, so a number of actions are available by default and in addition to this you can add your own actions if you like` (don't forget to add a period at the end :p)? There are no defaults.